.github/workflows/validate-yaml-manifests.yaml

```diff
@@ -0,0 +1,27 @@
+name: Lint
+
+# Trigger the workflow on push and pull request
+on: [push, pull_request]
+
+jobs:
+  yamllint:
+    name: Run yamllint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.22'
```

Comment:
See if you can reuse this - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/.github/workflows/weekly-security-scan.yaml#L20-L30